src/show.jl

```diff
@@ -31,18 +31,25 @@ function Base.show(io::IO, t::AbstractTerm)
     end
 end
 
+lessthanone(x::Real) = x < 0
+lessthanone(x::Complex) = false
+
+printcoeff(io::IO, coeff::Real, monomial) = print(io, abs(coeff) * monomial)
+printcoeff(io::IO, coeff::Complex, monomial) = print(io, "(", coeff, ")", monomial)
```

Comment:
printcoeff could also be used for terms to have a consistent printing between terms and polynomials.